src/components/LabelWarning.vue

```diff
@@ -0,0 +1,163 @@
+        return accumulator
+      }, [])
+    },
+    flowLabels() {
+      return this.flowGroup?.labels || this.flow?.environment?.labels
+    },
+    docsName() {
+      if (!this.agents?.length) return 'agents'
+      return 'Labels and Flow Affinity'
+    },
+    docsLink() {
+      if (!this.agents?.length)
+        return 'https://docs.prefect.io/orchestration/agents/overview.html'
+      return 'https://docs.prefect.io/orchestration/agents/overview.html#flow-affinity-labels'
+    },
+    labelsAlign() {
+      if (!this.agents?.length) {
+        this.labelMessage(
+          'You have no live Agents - scheduled flow runs will not be submitted for execution and will display as "Late".'
+        )
+        return false
+      }
+      if (
+        !this.flowLabels?.length &&
+        this.agentLabels.every(arrayOfLabels => arrayOfLabels.length > 0)
+      ) {
+        this.labelMessage(
+          'You have no Agents configured to pick up flows without labels; you may need to add labels to your flow.'
+        )
+        return false
+      } else {
+        let matchingLabels = 0
+        if (this.agentLabels) {
+          this.agentLabels.forEach(array => {
+            if (this.flowLabels.every(label => array.includes(label)))
+              matchingLabels++
+          })
+        }
+        if (matchingLabels > 0) {
+          this.labelMessage('Your flow and agent labels look good.')
+          return true
+        } else {
+          this.labelMessage(
+            'It looks like you have a mismatch of labels between your flow and running Agents. To allow an Agent to pick up this flow run, you need to have at least one Agent whose labels include those on the flow.'
+          )
+          return false
+        }
+      }
+    }
+  },
+  methods: {
+    labelMessage(message) {
+      this.infoMessage = message
+    }
+  }
+}
+</script>
+
+<template>
+  <v-menu
+    v-if="!labelsAlign"
+    :close-on-content-click="false"
+    offset-y
+    open-on-hover
+  >
+    <template v-slot:activator="{ on }">
+      <v-btn left x-small icon class="lefty" v-on="on">
+        <v-icon color="failRed">
+          fa-tag
+        </v-icon>
+        <v-icon
+          v-if="!labelsAlign"
+          class="position-absolute"
+          color="white"
+          size="x-small"
+        >
+          priority_high
+        </v-icon>
+      </v-btn>
+    </template>
+    <v-card tile class="pa-0" max-width="320">
+      <v-card-title class="subtitle pb-1">{{ agentOrLabel }} </v-card-title>
+
+      <v-card-text class="pt-0">
+        <div class="font-weight-bold pb-4"> {{ infoMessage }}</div>
+        <div
+          >You can see your agent labels in the
+          <router-link
+            target="_blank"
+            :to="{
+              name: 'dashboard',
+              params: { tenant: tenant.slug },
+              query: { agents: '' }
+            }"
+          >
+            <span>agents tab</span></router-link
+          ></div
```

Comment:
```suggestion
          >.</div
```
in case the diff is hard, this adds a period